src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  min-width: 1024px;
```

Comment:
you dont need this

Reply:
But without "display: flex" and "justify-content" don't work it. (I agree, min-width it's unecessary, but I thouth I need make as in Figma)